calculator.html

```diff
@@ -2,39 +2,40 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+    <link href="style.css" rel="stylesheet" />
     <title>Calculator</title>
   </head>
   <body>
     <form name="calculator">
       <table>
         <tr>
           <td colspan="4">
-            <input type="text" disabled>
+            <input type="text" id="display" disabled>
           </td>
         </tr>
         <tr>
-          <td><button type="button" value="1">1</td>
-          <td><button type="button" value="2">2</td>
-          <td><button type="button" value="3">3</td>
-          <td><button type="button" value="+">+</td>
+          <td><button class="box" type="button" value="1" onclick="calculator.display.value += '1'">1</td>
```

Comment:
Great job on the solution! I like how it's very simple! I would say that the follow-up work on this would be to extract the `onclick` functionality into an actual JS file which can be tested :)